benchmarks/StreamThroughput.cpp

```diff
@@ -178,8 +178,6 @@ void streamThroughput(unsigned, size_t items) {
   yarpl::Reference<BM_Subscriber> subscriber;
 
   BENCHMARK_SUSPEND {
-    LOG(INFO) << "  Running with " << items << " items";
```

Comment:
Given the recent changes in master, I think it's fine to leave this in.  Most users are going to use a high number for `items` so there won't be any more spammy lines of "Running with 1000 items".